controllers/imagehub/api/v1/image_webhook.go

```diff
@@ -59,9 +61,24 @@ func (m *ImageMutator) Default(ctx context.Context, obj runtime.Object) error {
 	img.ObjectMeta.Labels[SealosOrgLable] = img.Spec.Name.GetOrg()
 	img.ObjectMeta.Labels[SealosRepoLabel] = img.Spec.Name.GetRepo()
 	img.ObjectMeta.Labels[SealosTagLabel] = img.Spec.Name.GetTag()
+
+	if needMutate(img) {
+		oldimg := &Image{}
+		oldimg.Name = img.Name
+		err := m.Get(ctx, client.ObjectKeyFromObject(oldimg), oldimg)
+		if err != nil {
+			return client.IgnoreNotFound(err)
+		}
+		// mulate image cr
+		img.Spec.DetailInfo.Docs = oldimg.Spec.DetailInfo.Docs
```

Comment:
if the new cr has keyword had it is different will oldimg, what should I do?
Ignore all cr mutation or just ignore some?